test/JDBC/input/BABEL-3489-vu-verify.sql

```diff
@@ -0,0 +1,98 @@
+USE babel_3489_test_db;
```

Comment:
Can we add dependencies on this new function ?
Example View,Procedure referencing it